mappings/net/minecraft/world/level/LevelProperties.mapping

```diff
@@ -1,49 +1,35 @@
-CLASS net/minecraft/class_31 net/minecraft/world/level/LevelProperties
-	FIELD field_151 spawnX I
-	FIELD field_154 gameRules Lnet/minecraft/class_1928;
-	FIELD field_155 disabledDataPacks Ljava/util/Set;
-	FIELD field_156 customBossEvents Lnet/minecraft/class_2487;
-	FIELD field_157 difficultyLocked Z
-	FIELD field_158 version I
-	FIELD field_159 hardcore Z
-	FIELD field_162 versionName Ljava/lang/String;
-	FIELD field_165 lastPlayed J
-	FIELD field_167 spawnY I
-	FIELD field_168 thundering Z
-	FIELD field_169 levelName Ljava/lang/String;
-	FIELD field_170 playerData Lnet/minecraft/class_2487;
-	FIELD field_171 enabledDataPacks Ljava/util/Set;
-	FIELD field_172 playerDataLoaded Z
-	FIELD field_173 thunderTime I
-	FIELD field_174 difficulty Lnet/minecraft/class_1267;
-	FIELD field_176 clearWeatherTime I
-	FIELD field_17736 wanderingTraderSpawnDelay I
-	FIELD field_17737 wanderingTraderSpawnChance I
-	FIELD field_17738 wanderingTraderId Ljava/util/UUID;
-	FIELD field_177 commandsAllowed Z
-	FIELD field_179 gameMode Lnet/minecraft/class_1934;
-	FIELD field_181 sizeOnDisk J
-	FIELD field_182 spawnZ I
-	FIELD field_183 versionId I
-	FIELD field_184 dataFixer Lcom/mojang/datafixers/DataFixer;
-	FIELD field_185 initialized Z
-	FIELD field_189 time J
-	FIELD field_190 raining Z
-	FIELD field_191 scheduledEvents Lnet/minecraft/class_236;
-	FIELD field_192 rainTime I
-	FIELD field_193 worldData Ljava/util/Map;
-	FIELD field_194 versionSnapshot Z
-	FIELD field_196 dataVersion I
-	FIELD field_198 timeOfDay J
-	FIELD field_21837 serverBrands Ljava/util/Set;
-	FIELD field_21838 modded Z
-	FIELD field_23089 generatorOptions Lnet/minecraft/class_5285;
-	METHOD <init> (Lnet/minecraft/class_2487;Lcom/mojang/datafixers/DataFixer;ILnet/minecraft/class_2487;)V
+CLASS net/minecraft/class_5268 net/minecraft/world/level/LevelProperties
```

Comment:
Seriously, `ServerWorldProperties`, don't sneak in your trashy names

Reply:
problem is that now we have WorldProperties and LevelProperties describing the same construct. The impl of levelproperties is a big mess of interfaces like World, except iirc most of them aren't used anywhere.
Context is important!

Reply:
I'd say this specific category may work better in a specific pr we can bikeshed

Reply:
`WorldProperties` is the properties of a world, `LevelProperties` is `level.dat` and is both a `WorldProperties` and a `SaveProperties`. Simple as that.

Reply:
@i509VCB I've opened #1337 for debate, yet no one comes and just explains in this comment thread. I am pretty sure I've asked people to look into my debate issue yet no one seems to care yet, so :shrug: cannot do anything